apps/ocsp.c

```diff
@@ -1168,20 +1183,27 @@ static void make_ocsp_response(BIO *err, OCSP_RESPONSE **resp, OCSP_REQUEST *req
             X509 *ca_cert = sk_X509_value(ca, jj);
             OCSP_CERTID *ca_id = OCSP_cert_to_id(cert_id_md, NULL, ca_cert);
 
-            if (OCSP_id_issuer_cmp(ca_id, cid) == 0)
+            if (OCSP_id_issuer_cmp(ca_id, cid) == 0) {
                 found = 1;
-
+                if (resp_md != NULL && jj == 0) {
+                    OCSP_CERTID_free(cid_resp_md);
```

Comment:
Why is this free necessary? Didn't you already free this above?

Reply:
I wrote this so long ago... but, yeah, I don't think it's necessary.